tensorrt_llm/_torch/autotuner.py

```diff
@@ -13,31 +13,46 @@
 from tensorrt_llm.logger import logger
 
 
+@dataclass(frozen=True)
+class DynamicTensor:
+    tensor_idx: int
+    dim_idx: int
+    generating_rule: Union[Tuple[int], Callable]
+    mapping_rule: Callable
```

Comment:
```suggestion
class DynamicTensorSpec:
    input_idx: int
    dim_idx: int
    gen_tuning_buckets: Union[Tuple[int], Callable]
    map_to_tuning_buckets: Callable
```
Some suggestions for better naming. This class not a Tensor. It's more of a tuning spec.

Reply:
Thanks for the suggestions. Will apply it.